server/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "community-leaderboard",
+  "version": "1.0.0",
+  "description": "A community fitness leaderboard to help people keep a track of their physical activities and involve in friendly competition with their peers.",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "start": "node index.js",
+    "dev": "nodemon index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/NeerajHazarika/community-leaderboard.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/NeerajHazarika/community-leaderboard/issues"
+  },
+  "homepage": "https://github.com/NeerajHazarika/community-leaderboard#readme",
```

Comment:
Update the the git and bugs URL to point to https://github.com/dsc-iiest/community-leaderboard

and change license to MIT, which is what we have.

LGTM otherwise